src/pytorch_lightning/graveyard/core/__init__.py

```diff
@@ -0,0 +1,52 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any
+
+from pytorch_lightning import LightningDataModule, LightningModule
```

Comment:
couldn't we change these imports to directly import from core? since the imports only create references, it shouldn't matter where you actually import it from and the deeper we go with imports the less possibilites for circular import patterns we have

Reply:
This is fine because the graveyard is the last import at the root `__init__`